src/utils/people.ts

```diff
@@ -28,7 +29,7 @@ export async function handlePersonMessage(name: string, message: Message, client
         })
         let output = emojiStrings.join(" ");
         message.reply(output);
```

Comment:
you need to wrap lines 25 -> 31 in `if (emojiNames) { ... }` cuz there's no emojis for michael